File: s11_Retrofit/KriptoPare/app/src/main/java/com/omeryildizce/kriptopara/service/CryptoAPI.java
package com.omeryildizce.kriptopara.service;

import com.omeryildizce.kriptopara.model.CryptoModel;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;

public interface CryptoAPI {
    @GET("atilsamancioglu/K21-JSONDataSet/master/crypto.json")
    Call<List<CryptoModel>> getData();
}

File: s28_StateManagement/RetrofitCompose/app/src/main/java/com/omeryildizce/retrofitcompose/service/CryptoApi.kt
package com.omeryildizce.retrofitcompose.service

import com.omeryildizce.retrofitcompose.model.CryptoModel
import retrofit2.Call
import retrofit2.http.GET

interface CryptoApi {

    @GET("atilsamancioglu/K21-JSONDataSet/master/crypto.json")
    fun getData() : Call<List<CryptoModel>>

}